modules/oadp-backing-up-pvs-csi.adoc

```diff
@@ -27,7 +27,6 @@ metadata:
   labels:
     velero.io/csi-volumesnapshot-class: "true"
 driver: <csi_driver>
```

Comment:
Instead of removing it can we we add same thing we did in oadp-ceph-ceph-fs page 

deletionPolicy: <Retain_or_Delete> <1>
```suggestion
driver: <csi_driver>
```